tests/integration/test_sca/test_validate_remediation.py

```diff
@@ -0,0 +1,144 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
```

Comment:
```suggestion
copyright: Copyright (C) 2015-2023, Wazuh Inc.
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3566/commits/b91cb9df358e499995e51766152580067fb72776